client/keys/add.go

```diff
@@ -100,8 +105,30 @@ func runAddCmd(cmd *cobra.Command, args []string) error {
 			}
 		}
 
+		multisigKeys := viper.GetStringSlice(flagMultisig)
+		if len(multisigKeys) != 0 {
```

Comment:
`> 0`? Would it ever be negative? Should we return an error if negative?

Reply:
I doubt it: it returns a slice :)